src/goto-symex/goto_symex_state.h

```diff
@@ -141,6 +146,8 @@ class goto_symex_statet final : public goto_statet
   /// for error traces even after symbolic execution has finished.
   symbol_tablet symbol_table;
 
+  // Manager is requiered to be able to resize the thread vector
+  guard_managert guard_manager;
```

Comment:
We is it also being passed to all the `rename` functions when we have a guard manager here?